core/ir/x86/decode_table.c

```diff
@@ -7857,6 +7867,18 @@ const instr_info_t vex_W_extensions[][2] = {
   }, { /* vex_W_ext 109 */
     {OP_vmovd, 0x660f7e10, "vmovd", Ed, xx, Vd_dq, xx, xx, mrm|vex, x, tevexwb[136][0]},
     {OP_vmovq, 0x660f7e50, "vmovq", Ey, xx, Vq_dq, xx, xx, mrm|vex, x, tevexwb[136][2]},
+  }, { /* vex_W_ext 110 */
+    {OP_vpdpbusd, 0x66385008, "vpdpbusd", Ve, xx, KEd, He, We, mrm|evex|ttfvm, x, tevexwb[267][0]},
```

Comment:
OK, let's see if I can get the details right here:

The manual says:
```
VEX.128.66.0F38.W0 50 /r     VPDPBUSD xmm1, xmm2, xmm3/m128
VEX.256.66.0F38.W0 50 /r     VPDPBUSD ymm1, ymm2, ymm3/m256
```

VEX means it can be encoded with either vex or evex IIRC?  So this entry should not have `|evex` but rather `|vex`?

"W0" means evex.W has to be 0.  The W=1 entries here are all invalid so that makes sense.

Since it has both 128 and 256, evex.L will decide which (1==256).
So the 256 versions should all have 0x80 set in the bottom byte to indicate the L must be set?
No, I think we're relying on the operands to set evex.L (and the reverse for decoding) for these cases: those aren't different table entries.

There is no broadcast version so no evex.B.

The 0F38 should set OPCODE_THREEBYTES so bottom bits 0x8.

No disp compression and no broadcast: I think that's DR_TUPLE_TYPE_FVM?  I wish there were comments reminding us what these confusing DR tuple type enum names mean (if you want to add that sounds good to me).